### PR TITLE
Labchart no time

### LINKED ISSUE #138: Error when there is no 'time' column (for labchart txt files)
When using phys2bids with a labchart txt file that does _not_ have time as the first column, the file is not correctly processed. 

## Expected Behavior

When I run:

`phys2bids -in Test2_trigger_CO2_O2_pulse_1000Hz_534TRs.txt -info -chtrig 0`

I expect:

```
File extension is .txt
Reading the file
Reading infos
File Test2_trigger_CO2_O2_pulse_1000Hz_534TRs.txt contains:

00. CO2; sampled at 1000.0 Hz
01. O2; sampled at 1000.0 Hz
02. Pulse; sampled at 1000.0 Hz
saving channels plot at plot at Test2_trigger_CO2_O2_pulse_1000Hz_534TRs.png
```

I expect the plot to show: trigger, CO2, O2 and pulse. 

## Actual Behavior

This is what I get:

```
File extension is .txt
Reading the file
Reading infos
File Test2_trigger_CO2_O2_pulse_1000Hz_534TRs.txt contains:

00. Trigger; sampled at 1000.0 Hz
01. CO2; sampled at 1000.0 Hz
saving channels plot at plot at Test2_trigger_CO2_O2_pulse_1000Hz_534TRs.png
```

Test2_trigger_CO2_O2_pulse_1000Hz_534TRs

## Steps to Reproduce the Problem

Run the command I ran above. The data can be found on OSF/labchart/chicago
    
## Specifications

Python version: 3.7
phys2bids version: phys2bids v1.1.0-beta+236.g3a6fc72
Platform: MacOS

## Possible solution
I am not sure. It seems to be plotting time as the first column, even though I specified that trigger was '0'. Then I am unsure what the second and third are!

### PR DESCRIPTION
**Pull request by vinferrer**
Closes #138 

## Proposed Changes

  - Now labchart files without time channel are accepted 
  - Plus test_txt.py downloads the testing files directly from OSF

### CODE REVIEW

**Comment by vinferrer** [comment]
Guys, I need to add wget package for the test since I use it to dowload files from OSF, Could you tell me the files i have to modify?

**Comment by vinferrer** [comment]
solved

**Comment by eurunuela** [comment]
Seeing that the bug in #138 seems to be fixed, is this ready to review @vinferrer ?

File: phys2bids/interfaces/txt.py [hunk 1]
```diff
@@ -68,21 +68,31 @@ def process_labchart(channel_list, chtrig, header=[]):
...
-    names = names + orig_names
     # get channels
     timeseries = np.matrix(channel_list).T.tolist()
     freq = [1 / interval[0]] * len(timeseries)
     timeseries = [np.array(darray) for darray in timeseries]
-    ordered_timeseries = [timeseries[0], timeseries[chtrig]]
-    timeseries.pop(chtrig)
-    timeseries.pop(0)
-    ordered_timeseries = ordered_timeseries + timeseries
+    if (orig_names_len < len(timeseries)):
```
Comment (eurunuela) [inline phys2bids/interfaces/txt.py]: Could you please add a comment to what each of the if statements is doing?

File: phys2bids/tests/test_txt.py [hunk 2]
```diff
@@ -4,40 +4,42 @@
 from pytest import raises
 import sys
 import io
+import wget
 
 
 def test_read_header_and_channels():
-    test_filename = 'Test_belt_pulse_samefreq.txt'  # this file has a comment and extra tab
+    url = 'https://osf.io/sdz4n/download'
```
Comment (eurunuela) [inline phys2bids/tests/test_txt.py]: Could you please add a comment that states this is the url to the file `test_filename`?

File: phys2bids/tests/test_txt.py [hunk 3]
```diff
@@ -4,40 +4,42 @@
...
+    wget.download(url, test_full_path)
     chtrig = 2
     header, channels = txt.read_header_and_channels(test_full_path, chtrig)
     assert len(header) == 16  # check proper header lenght
     assert len(channels) == 1336816  # check proper number of timepoints
     assert len(header[-1]) == 6  # check extra line is deleted
     # load file with comment
-    assert len(channels[6]) == 6  # check the comment has been eliminated
-    test_filename = 'Test_2minRest_trig_multifreq_header_comment.txt'  # this file has a comment and extra tab
+    url = 'https://osf.io/q4x2f/download'
```
Comment (eurunuela) [inline phys2bids/tests/test_txt.py]: Same as before, please comment the url corresponds to `test_filename`.

File: phys2bids/tests/test_txt.py [hunk 4]
```diff
@@ -4,40 +4,42 @@
...
 
 
 def test_populate_phys_input():
-    # testing error for files without header
-    test_filename = 'Test_belt_pulse_samefreq_no_header.txt'
-    test_path = resource_filename('phys2bids', 'tests/data')
-    test_full_path = os.path.join(test_path, test_filename)
-    chtrig = 2
     # testing for AcqKnoledge files
     test_filename = 'Test_belt_pulse_samefreq.txt'
```
Comment (eurunuela) [inline phys2bids/tests/test_txt.py]: Please comment that the file has already been downloaded and no more downloading is required.

File: phys2bids/interfaces/txt.py (hunk 1 above)
Comment (vinferrer) [reply phys2bids/interfaces/txt.py]: done

File: phys2bids/tests/test_txt.py (hunk 4 above)
Comment (vinferrer) [reply phys2bids/tests/test_txt.py]: done

File: phys2bids/tests/test_txt.py (hunk 3 above)
Comment (vinferrer) [reply phys2bids/tests/test_txt.py]: done

**Pull request merged**